api.go

```diff
@@ -1112,6 +1112,33 @@ func (api *API) ImportValue(ctx context.Context, req *ImportValueRequest, opts .
 	return errors.Wrap(err, "importing")
 }
 
+func (api *API) ImportColumnAttrs(ctx context.Context, req *ImportColumnAttrsRequest, opts ...ImportOption) error {
+	span, _ := tracing.StartSpanFromContext(ctx, "API.ImportColumnAttrs")
+	defer span.Finish()
+
+	index, err := api.Index(ctx, req.Index)
+	if err != nil {
+		return errors.Wrap(err, "getting index")
+	}
+
+	if err := api.validateShardOwnership(req.Index, uint64(req.Shard)); err != nil {
+		return errors.Wrap(err, "validating shard ownership")
+	}
+
+	bulkAttrs := make(map[uint64]map[string]interface{})
+	for n := 0; n < len(req.ColumnIDs); n++ {
+		bulkAttrs[uint64(n)] = map[string]interface{}{req.AttrKey: req.AttrVals[n]}
```

Comment:
Why the map of sequential values?  Could a slice of maps be used in place? I am naive on the attribute storage, just curious